build.sbt

```
@@ -112,7 +112,18 @@ lazy val model = (projectMatrix in file("model"))
     ): _*
   )
   .settings(
-    jacocoReportSettings := jacocoSettings(scalaVersion.value, "atum-agent: model"),
+    jacocoReportSettings := jacocoSettings(scalaVersion.value, "atum-model"),
     jacocoExcludes := jacocoProjectExcludes()
   )
-  .jvmPlatform(scalaVersions = Versions.clientSupportedScalaVersions)
+  .jvmPlatform(scalaVersions = Versions.supportedScalaVersions)
```

```suggestion
  .jvmPlatform(scalaVersions = Versions.serviceScalaVersion)
```

This local issue is blocking built. Prepare proposal how to solve